FreeRTOS/FreeRTOS
# Add tests for the length caps within xStreamBufferSend().

Description
-----------
Add tests for the changes in this PR https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/195

Test Steps
-----------
Run a demo that builds both FreeRTOS/Demo/Common/Minimal/StreamBufferDemo.c and FreeRTOS/Demo/Common/Minimal/MessageBufferDemo.c

Related Issue
-----------
https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/195


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
